spec/controllers/publish_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.describe PublishController do
+  describe '#submitting_pages_not_present?' do
+    before do
+      allow(controller).to receive(:service).and_return(service)
+    end
+    let(:service) { MetadataPresenter::Service.new(metadata_fixture(:branching)) }
```

Comment:
```suggestion
    let(:service) { MetadataPresenter::Service.new(latest_metadata) }
    let(:latest_metadata { metadata_fixture(:branching) }
```

then you can remove lines 25 and 39. just a thought